### PR TITLE
fuzzgen: Add more opcodes

### PR DESCRIPTION
**Pull request by afonso360**
👋 Hey,

This add a few more opcodes to fuzzgen.

I've fuzzed this for a couple of hours on x86_64 and it's stable, it's also stable on AArch64 with #5118 applied.

### CODE REVIEW

**Review by jameysharp — APPROVED** [review]
Looks good to me!

Should wait for #5118 to merge before merging this? It's not strictly necessary since OSS-Fuzz doesn't run on aarch64, but I guess this change is inconvenient for anyone who's routinely fuzzing Cranelift on aarch64. On the other hand, I don't know of anyone besides you who's doing that, so I guess this decision is up to you!

**Comment by afonso360** [comment]
I don't mind waiting for https://github.com/bytecodealliance/wasmtime/pull/5118 before merging this.

**Pull request merged**
